schemas/session-replay/mobile/placeholder-wireframe-schema.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "session-replay/mobile/placeholder-wireframe-schema.json",
+  "title": "PlaceholderWireframe",
+  "type": "object",
+  "description": "Schema of all properties of a PlaceholderWireframe.",
+  "allOf": [
+    {
+      "$ref": "_common-wireframe-schema.json"
+    },
+    {
+      "required": ["type"],
+      "properties": {
+        "type": {
+          "type": "string",
```

Comment:
Because you need agreement between recorders & player on which placeholder types are supported, I'd rather make this an `enum` [like we do for error types](https://github.com/DataDog/rum-events-format/blob/c58d69273d8a79a7443407310ec9a458ab1075a5/schemas/rum/error-schema.json#L42).